auctioneer/program/tests/execute_sale.rs

```diff
@@ -536,6 +548,13 @@ async fn execute_sale_two_bids_success() {
         .unwrap();
     assert_eq!(buyer1_token_before.is_none(), true);
 
+    let listing_config_account = context
+          .banks_client
+          .get_account(listing_config_address)
+          .await
+          .unwrap()
```

Comment:
Awesome! So last request. We know that if this block is called after the account is closed (Line 558) that the first unwrap returns an Option that is None. So the last check we need is something at the very end that retrieves this account again, unwraps it once, and checks and verifies that Option == None and the account is officially closed.

Reply:
Done!